### ISSUE TITLE
Question about input data normalization  [mean,std] value  where come from

### ISSUE THREAD

**Issue by zkailinzhang** [Xiaoccer/ReID-PCB_RPP#4]
https://github.com/Xiaoccer/ReID-PCB_RPP/blob/00462d044864f23581820b9696ca02afe04391e7/train.py#L74
and the val data  in line 79
is same ?

**Comment by Xiaoccer** [comment]
I use the pre-trained model.
https://github.com/pytorch/examples/blob/5df464c46cf321ed1cc3df1e670358d7f5ae1887/imagenet/main.py#L198
